BraveRewardsUI/Tipping/TippingOverviewView.swift

```diff
@@ -48,9 +48,13 @@ class TippingOverviewView: UIView {
     $0.spacing = 10.0
   }
   
-  let disclaimerView = DisclaimerView(text: Strings.TippingUnverifiedDisclaimer).then {
-    $0.isHidden = true
+  let disclaimerView = LinkLabel().then {
+    $0.textColor = Colors.grey200
+    $0.font = UIFont.systemFont(ofSize: 12.0)
+    $0.textContainerInset = UIEdgeInsets(top: 8.0, left: 8.0, bottom: 8.0, right: 8.0)
+    $0.text = "\(Strings.TippingUnverifiedDisclaimer) \(Strings.DisclaimerLearnMore)"
     $0.backgroundColor = .white
+    $0.isHidden = true
```

Comment:
Missing background color + corner radius as. it was before:

https://github.com/brave/brave-rewards-ios/blob/a9a15960f87b3577c6db036519782d5b4eebbced/BraveRewardsUI/Common/DisclaimerView.swift#L37-L38

If this isn't possible to fix, perhaps we revert the deletion of `DisclaimerView` and simply embed a `LinkLabel` within it?